src/containers/CustomFunctionStep.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { t } from 'ttag'
+import { connect, ConnectedProps } from 'react-redux'
+import ConfigurationStep from './ConfigurationStep'
+import CustomFunctions from './CustomFunctions'
+
+const connector = connect(({ runConfiguration }: { runConfiguration: any }) => {
+  const { method } = runConfiguration
+
+  return { method }
+})
+
+type CustomFunctionProps = ConnectedProps<typeof connector> & {
+  number: number
+}
+
+const CustomFunctionStep = ({ number, method }: CustomFunctionProps) => {
+  if (method !== 'function') {
+    return null
+  }
+
+  return (
+    <ConfigurationStep title={t`Select functions`} number={number} name="functions" active>
+      <CustomFunctions />
+    </ConfigurationStep>
+  )
+}
+
+CustomFunctionStep.shouldRender = ({ runConfiguration }: { runConfiguration: any }) =>
+  runConfiguration.method === 'function'
```

Comment:
I couldn't figure out what `shouldRender` does, but thought it might be adding this since there is similar code in `TraitStep`.

Reply:
Yep, that's correct. Basically it's "should we display this step?"

Reply:
It doesn't look like there's any code (in any consbio repo) that makes use of `shouldRender`: https://github.com/search?q=org%3Aconsbio+shouldRender&type=code